tests/unit-tests/reporter/models/reportsManager-test.js

```diff
@@ -379,6 +379,45 @@ describe('Reports manager tests', function () {
             should.exist(reports);
             reports.length.should.eql(0);
         });
+
+        it('get last report with avg rsp when test running', async () => {
+            const now = new Date();
+            const tenSecBefore = new Date(now).setSeconds(now.getSeconds() - 10);
+            const subscriber = { last_stats: { rps: { total_count: 200 }, codes: { '200': 10 } } };
+            const report = Object.assign({}, REPORT, { last_updated_at: now, start_time: tenSecBefore, subscribers: [subscriber] });
+            databaseGetLastReportsStub.resolves([report]);
+            const reports = await manager.getLastReports();
+            reports.length.should.eql(1);
+            should(reports[0].avg_rps).eql(20);
+        });
+        it('get last report with avg rsp when test finished', async () => {
```

Comment:
add another unit test case when test run is aborted

Reply:
fixed